deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -1,25 +1,27 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-bases:
-- ../../base
+namespace:
+  gce-pd-csi-driver
 images:
-- name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
   # Don't change stable image without changing pdImagePlaceholder in
```

Comment:
Keep this comment with gcp-compute-persistent-disk-csi-driver

Reply:
fixed